moderncvbodyv.sty

```diff
@@ -67,12 +72,21 @@
   % regular lengths
   \setlength{\parskip}{0\p@}}
 
+\newcommand{\calculatecompensation}[1]{%
+  % calculate the number of lines used by sections and subsections
+  \settowidth{\thissectionwidth}{#1}%
+  \renewcommand{\numoflines}{\numexpr\thissectionwidth/\hintscolumnwidth\relax}%
```

Comment:
It seems this might break in case section name contains line breaks (?).

Reply:
Uh, I did not test this, but you may be right. I will catch up on that test first :+1: